release-notes/9.0/preview/preview6/api-diff/Microsoft.NETCore.App/9.0-preview6_System.Runtime.Intrinsics.Arm.md

```diff
@@ -0,0 +1,1058 @@
+# System.Runtime.Intrinsics.Arm
```

Comment:
System.Runtime.Intrinsics.Arm changes LGTM.

CC. @kunalspathak, @dotnet/area-system-runtime-intrinsics 

Notably the name changes in AdvSimd were for new APIs introduced this release, API review opted for a different format to make things overall more consistent with `Vector<T>`